feature-labs/pom.xml

```diff
@@ -0,0 +1,119 @@
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>${maven-resources-plugin.version}</version>
+                <inherited>false</inherited>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.basedir}/target</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>src/main/resources</directory>
+                                    <filtering>true</filtering>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.karaf.tooling</groupId>
+                <artifactId>karaf-maven-plugin</artifactId>
+                <version>${karaf.version}</version>
+                <executions>
+                    <execution>
+                        <id>add-features-to-repo</id>
+                        <phase>process-test-resources</phase>
+                        <goals>
+                            <goal>features-add-to-repository</goal>
+                        </goals>
+                        <configuration>
+                            <descriptors>
+                                <descriptor>file:${basedir}/target/feature-labs.xml</descriptor>
+                            </descriptors>
+                            <features>
+                                <feature>jclouds-azurecompute-arm</feature>
+                                <feature>jclouds-b2</feature>
+                                <feature>jclouds-cloudsigma2-hnl</feature>
+                                <feature>jclouds-cloudsigma2-lvs</feature>
+                                <feature>jclouds-cloudsigma2-sjc</feature>
+                                <feature>jclouds-cloudsigma2-wdc</feature>
+                                <feature>jclouds-cloudsigma2-zrh</feature>
+                                <feature>jclouds-google-cloud-storage</feature>
+                                <feature>jclouds-packet</feature>
+                                <feature>jclouds-vagrant</feature>
+                            </features>
+                            <repository>target/features-repo</repository>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
That's probably a historic baggage. All that's needed in a feature pom is:
```
    <artifactId>...</artifactId>
    <name>.....</name>
    <packaging>feature</packaging>

    <build>
        <plugins>
            <plugin>
                <groupId>org.apache.karaf.tooling</groupId>
                <artifactId>karaf-maven-plugin</artifactId>
                <version>${karaf.plugin.version}</version>
                <extensions>true</extensions>
            </plugin>
        </plugins>
    </build>
```

To verify it's still working after the change make sure you still have the following two in your logs:
```
[INFO] Installing .../jclouds-karaf/feature-labs/pom.xml to /Users/svet/.m2/repository/org/apache/jclouds/karaf/jclouds-karaf-labs/2.1.0-SNAPSHOT/jclouds-karaf-labs-2.1.0-SNAPSHOT.pom
[INFO] Installing .../jclouds-karaf/feature-labs/target/feature-labs.xml to /Users/svet/.m2/repository/org/apache/jclouds/karaf/jclouds-karaf-labs/2.1.0-SNAPSHOT/jclouds-karaf-labs-2.1.0-SNAPSHOT-features.xml
```


Same for the non-labs feature pom.